test.env.example

```diff
@@ -0,0 +1,11 @@
+SNOWFLAKE_TEST_ACCOUNT=<account>
+SNOWFLAKE_TEST_ALT_DATABASE=<database_name_in_alt_warehouse>
```

Comment:
```suggestion
SNOWFLAKE_TEST_ALT_DATABASE=<name of database you must create; e.g. DBT_SF_TEST_ALT>
```

Reply:
Nit: "name of existing database"

Reply:
Am I correct in saying that "you must create" == "already existing"

Reply:
That's right. I appreciate that's a more succinct way to capture that.